src/src/ESPEasyCore/ESPEasyRules.cpp

```diff
@@ -480,9 +480,9 @@ void parse_string_commands(String& line) {
     // Command without opening and closing brackets.
     const String fullCommand = line.substring(startIndex + 1, closingIndex);
     const String cmd_s_lower = parseString(fullCommand, 1, ':');
-    const String arg1        = parseStringKeepCase(fullCommand, 2, ':');
-    const String arg2        = parseStringKeepCase(fullCommand, 3, ':');
-    const String arg3        = parseStringKeepCase(fullCommand, 4, ':');
+    const String arg1        = parseStringKeepCaseNoTrim(fullCommand, 2, ':');
```

Comment:
This does change the functionality, probably not intended....

Reply:
It does enable/allow the use of leading & trailing spaces in these functions...